docker-compose.yaml

```diff
@@ -64,7 +64,7 @@ services:
         condition: service_completed_successfully
 
   kakarot-rpc:
-    image: ghcr.io/kkrt-labs/kakarot-rpc/node:latest
+    image: eliastazartes/kakarot-rpc:v0.5.0
```

Comment:
ghcr.io/kkrt-labs/kakarot-rpc/node:latest?

Reply:
ok! but we need to release

Reply:
I can do now then! Will just merge apoorv's PR